packages/syft/src/syft/node/credentials.py

```diff
@@ -53,10 +54,9 @@ def __hash__(self) -> int:
 class SyftSigningKey(SyftBaseModel):
     signing_key: SigningKey
 
-    def __init__(self, signing_key: Union[str, SigningKey]) -> None:
-        if isinstance(signing_key, str):
-            signing_key = SigningKey(bytes.fromhex(signing_key))
-        super().__init__(signing_key=signing_key)
+    @pydantic.validator("signing_key", pre=True, always=True)
+    def make_verify_key(cls, v: Union[str, SigningKey]) -> SigningKey:
```

Comment:
NITPICK: the method name is make_signing_key?

Reply:
Fixed by: https://github.com/OpenMined/PySyft/pull/7665/commits/d22ffe63abbfc2d76c8dcdfa162c474e2af0ca8d